src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -218,7 +218,14 @@ private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootL
             }
         }
 
-        private JToken TransformContentCore(ErrorBuilder errors, JsonSchemaDefinition definitions, FilePath file, JsonSchema schema, JToken token, int uidCount)
+        private JToken TransformContentCore(
+            ErrorBuilder errors,
+            JsonSchemaDefinition definitions,
+            FilePath file,
+            JsonSchema schema,
+            JToken token,
+            int uidCount,
+            bool contentFallback = true)
```

Comment:
remove the default value (`=true`) to ensure `LoadXrefProperty` method also call this method with the correct value.

Reply:
Sure.

---
In reply to: [489235692](https://github.com/dotnet/docfx/pull/6558#discussion_r489235692) [](ancestors = 489235692)